### PR TITLE
17 tariff group for user

### PR DESCRIPTION
**Pull request by romigos**
begin to make user

### CODE REVIEW

File: app/models/tariff_group.rb [hunk 1]
```diff
@@ -12,4 +12,5 @@
 
 class TariffGroup < ApplicationRecord
   has_many :tariffs
+  has_one :user
```
Comment (saviokmua) [inline app/models/tariff_group.rb]: @romigos has_many :users

File: db/migrate/20190122220827_add_nameadmin_tariff_id_to_user.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddNameadminTariffIdToUser < ActiveRecord::Migration[5.2]
```
Comment (saviokmua) [inline db/migrate/20190122220827_add_nameadmin_tariff_id_to_user.rb]: `class AddNameAdminTariffIdToUser < `

**Review by saviokmua — CHANGES_REQUESTED** [review]

File: db/migrate/20190122220827_add_nameadmin_tariff_id_to_user.rb (hunk 2 above)
Comment (romigos) [reply db/migrate/20190122220827_add_nameadmin_tariff_id_to_user.rb]: fixed

File: app/models/tariff_group.rb (hunk 1 above)
Comment (romigos) [reply app/models/tariff_group.rb]: fixed

**Pull request merged**